### PR TITLE
Set SGX_MODE Default

### PR DESCRIPTION
**Pull request by danintel**
Set SGX_MODE default to `SIM`. This makes SGX_MODE optional.

Signed-off-by: danintel <daniel.anderson@intel.com>

### CODE REVIEW

File: examples/enclave_manager/setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 module_path = 'tc/sgx/trusted_worker_manager/enclave_wrapper'
 module_src_path = os.path.join(tcf_root_dir, module_path)
 
-sgx_mode_env = os.environ.get('SGX_MODE', None)
+sgx_mode_env = os.environ.get('SGX_MODE', 'SIM')
```
Comment (manju956) [inline examples/enclave_manager/setup.py]: Could you please add similar changes in setup.py in below path ?
examples/common/python/setup.py

Comment (danintel) [reply examples/enclave_manager/setup.py]: I already do that.  See the change above in `examples/common/python/setup.py`:
```
- sgx_mode_env = os.environ.get('SGX_MODE', None)
+ sgx_mode_env = os.environ.get('SGX_MODE', 'SIM')
```

**Comment by danintel** [comment]
@manju956: look again. I set `SGX_MODE` in both Python `setup.py` files, 
`examples/common/python/setup.py` and `examples/enclave_manager/setup.py`

**Comment by manju956** [comment]
> @manju956 : look again. I set `SGX_MODE` in both Python `setup.py` files,
> `examples/common/python/setup.py` and `examples/enclave_manager/setup.py`

My bad, missed that file.

File: docker-compose.yaml [hunk 2]
```diff
@@ -34,6 +34,8 @@ services:
       - /tmp/.X11-unix:/tmp/.X11-unix
       - ~/.Xauthority:/root/.Xauthority
     working_dir: "/project/TrustedComputeFramework/tools/build"
+    environment:
```
Comment (arsulegai) [inline docker-compose.yaml]: Is this required, given that the setup.py, cmake/make files are assuming the default value to be SIM?

Comment (danintel) [reply docker-compose.yaml]: @arsulegai. No this is not required. It makes it obvious for anyone using this script that it is for SGX simulator mode use only.

**Pull request merged**